README.md

```diff
@@ -69,7 +81,7 @@ Now running `yarn build` will do the same thing as `yarn calliope build`. You ca
 
 # Configuration
 
-Calliope works without additional configuration. Should you need to configure how any of the default tasks behave or add configuration for a custom task, you’ll be able to do so by creating a `calliope.config.js` file in the same directory as your project’s `package.json`. See the [`calliope.sample-config.js`] file in this repository for details on how to configure each individual task from your newly-created configuration file, or as a reference for creating your configuration object for a custom task.
+Calliope works without additional configuration. Should you need to configure how any of the default tasks behave or add configuration for a custom task, you’ll be able to do so by creating a `calliope.config.js` file in the same directory as your project’s `package.json`. See the [`boilerplate/calliope.config-sample.js`] file in this repository for details on how to configure each individual task from your newly-created configuration file, or as a reference for creating your configuration object for a custom task.
```

Comment:
This change (and similar in this file) is part of standardizing the way boilerplate files are named, which now follows the `FILENAME-sample.EXT` format.